src/test/java/com/ericsson/ei/flowtests/FlowTestBase.java

```diff
@@ -245,12 +245,14 @@ private long countProcessedEvents(String database, String collection) {
 
     protected void waitForEventsToBeProcessed(int eventsCount) {
         // wait for all events to be processed
+        long stopTime = System.currentTimeMillis() + 30000;
         long processedEvents = 0;
-        while (processedEvents < eventsCount) {
+        while (processedEvents < eventsCount && stopTime > System.currentTimeMillis()) {
             processedEvents = countProcessedEvents(database, event_map);
             LOGGER.info("Have gotten: " + processedEvents + " out of: " + eventsCount);
+            System.out.println("Have gotten: " + processedEvents + " out of: " + eventsCount);
```

Comment:
Is this just for test debugging,, change to a LOGGER message if it is going to be kept.

Reply:
Done.